src/logic/map_objects/tribes/production_program.cc

```diff
@@ -1343,16 +1404,16 @@ void ProductionProgram::ActMine::execute(Game& game, ProductionSite& ps) const {
 		} while (mr.advance(*map));
 	}
 
-	//  how much is digged
-	int32_t digged_percentage = 100;
+	//  how much is dug
+	unsigned dug_percentage = MapObjectProgram::kMaxProbability;
```

Comment:
`10000` corresponds to `100%` now, because we represent floating point as int. E.g. `33.33%` is represented as `3333`.

I have printed some test log output and tested this to verify that my scaling is correct.